src/assets/styles/_helper.scss

```diff
@@ -1,1726 +1,17 @@
+@import "align";
+@import "background";
+@import "border";
 @import "colours";
+@import "display";
+@import "float";
+@import "form";
 @import "loader";
+@import "margin";
+@import "misc";
+@import "overflow";
+@import "padding";
+@import "position";
+@import "status";
+@import "text";
 @import "width-height";
 @import "z-index";
-
-body,
-html {
-  width: 100%;
-  height: 100%;
-  position: relative;
-  overflow: hidden;
-  -webkit-tap-highlight-color: transparent;
-  min-height: 100%;
-  -webkit-font-smoothing: antialiased;
-}
-
-html {
-  font-size: 16px;
-}
-
-body {
-  margin: 0;
-  padding: 0;
-  font-weight: 400;
-  font-size: 0.875rem;
-  color: rgba(0, 0, 0, 0.87);
-  line-height: 1.5;
-  font-family: Roboto, Helvetica Neue, sans-serif;
-
-  table {
-    border: 1px solid rgba($color: #000000, $alpha: 0.12);
-    width: 100%;
-    tr:nth-child(even) {
-      background-color: #f2f2f2;
-    }
```

Comment:
I am not able to locate this in any file. Can you please check for all such differences in the files and ensure nothing is missing? Also, make sure if anything was nested it remains nested.